CONTRIBUTING.md

```diff
@@ -87,6 +87,9 @@ usual things:
 *   Identify potential functional problems.
 *   Identify code duplication.
 *   Ensure the unit tests have enough coverage.
+*   Ensure continuous integration (CI) bots run on the PR. If not run (in the
+    case of PRs by external contributors), add the "kokoro:run" label to the
```

Comment:
I thought only project members can add the label?  I think this needs to be rephrased to *request* a project maintainer to add the label.

Reply:
You're right. But I have added these new lines in the section titled
#### For maintainers: Reviewing a PR

Reply:
Bah.  My bad. I didn't see the context.